src/hub/ShareClassManager.sol

```diff
@@ -35,31 +36,29 @@ contract ShareClassManager is Auth, IShareClassManager {
 
     IHubRegistry public hubRegistry;
```

Comment:
Personally think we should make this `immutable`, would you agree @wischli @mustermeiszer ?